webapp/src/components/AssetPage/CategoryBadge/CategoryBadge.tsx

```diff
@@ -2,13 +2,32 @@ import React from 'react'
 import { t } from 'decentraland-dapps/dist/modules/translation/utils'
 import IconBadge from '../IconBadge'
 import { Props } from './CategoryBadge.types'
+import { getSearchWearableSection } from '../../../modules/routing/search'
+import { locations } from '../../../modules/routing/locations'
```

Comment:
We have some (annoying maybe :P) [import order standards](https://www.notion.so/decentraland/dApps-Standards-760ee6551c9f4c00aa71f7d9c50a7646#8d48e37938124a41ac664112cb1e27f6) we should follow!

```suggestion
import { t } from 'decentraland-dapps/dist/modules/translation/utils'
import { getSearchWearableSection } from '../../../modules/routing/search'
import { locations } from '../../../modules/routing/locations'
import IconBadge from '../IconBadge'
import { Props } from './CategoryBadge.types'
```

Reply:
Many thanks for remember me this